test/mocha/core/curlify.js

```diff
@@ -194,6 +194,54 @@ describe("curlify", function() {
         expect(curlified).toEqual("curl -X POST \"http://example.com\" -H  \"content-type: multipart/form-data\" -F \"id=123\" -F \"file=@file.txt;type=text/plain\"")
     })
 
+    it("should print a curl with formData and file, and trailing custom header", function () {
+        // use case: requestInterceptor appends req.headers as trailing header
```

Comment:
Shouldn't use comments, unless absolutely necessary. Everything should be translated into test descriptions.